BUILD

```diff
@@ -36,17 +36,36 @@ this file by hand, before proceeding with the build. Note that the the generated
 a symbolic link to a system-specific default config file. It is not necessary to edit the file; you
 can override variables on the "make" command line if you prefer.
 
-An alternative to "make mconfig" is to use the provided "configure" script. It will try to
-generate a suitable "mconfig" file, based on sensible defaults and options provided on the command
-line when the script is run.
+An alternative to "make mconfig" is to use the provided "configure" script. See "configure script",
+below.
+
+You can also create and edit the "mconfig" file completely by hand (or start by copying one for a
+particular OS from the "configs" directory) to choose appropriate values for the configuration
+variables defined within.
+
+
+"configure" script
+=-=-=-=-=-=-=-=-=-
+
+"configure" is a script to generate a suitable "mconfig" file, based on sensible defaults and
+options provided on the command line when the script is run. It also has checks to enable/disable
+certain features based on the availability of their requirements (such as presence of certain
+header files).
 
 For more information on available options from the configure script, run:
 
     ./configure --help
 
-You can also create and edit the "mconfig" file completely by hand (or start by copying one for a
-particular OS from the "configs" directory) to choose appropriate values for the configuration
-variables defined within.
+If you are building Dinit for another platform (i.e. cross-compiling), one of the checks performed
+by "configure" in native builds is the "-fno-rtti" check which is done by compiling a test program
+and running it to ensure "-fno-rtti" compiler option works correctly. However, because of the nature
+of cross-compiling, it's not possbile to compile and run the test program on the building machine,
+and instead, "configure" disables "-fno-rtti" in this case and prints out this warning:
```

Comment:
The sentence is too long, it should end just after "it's not possbile to compile and run the test program on the building machine" and continue as a new sentence (without "and").